### PR TITLE
TASK-3841 - IVA REST_APIapp for /analysis/alignment/coverage/qc/geneCoverageStats/run is missing the field "genes"

### PR DESCRIPTION
**Pull request by Rodielm**
This PR fix genes field

### CODE REVIEW

**Comment by Rodielm** [comment]
Task linked: [TASK-3841 IVA REST_APIapp for /analysis/alignment/coverage/qc/geneCoverageStats/run is missing the field "genes"](https://app.clickup.com/t/36631768/TASK-3841)

File: src/webcomponents/api/rest-endpoint.js [hunk 1]
```diff
@@ -105,6 +108,7 @@ export default class RestEndpoint extends LitElement {
         this.result = "";
         const isPrimitiveOrEnum = dataParameter => !dataParameter.complex || dataParameter.type === "enum";
         const isObject = dataParameter => dataParameter.complex && UtilsNew.isNotEmptyArray(dataParameter?.data);
+        const isListString = dataParameter => dataParameter.complex && (dataParameter.type === "List") && (dataParameter["data"] === undefined);
```
Comment (jmjuanes) [inline src/webcomponents/api/rest-endpoint.js:111]: Please use `typeof dataParameter["data"] === "undefined"` instead.

File: src/webcomponents/api/rest-endpoint.js [hunk 2]
```diff
@@ -151,6 +155,13 @@ export default class RestEndpoint extends LitElement {
                         if (isObject(dataParameter)) {
                             bodyElements.push(this.buildObjectOrListForm(dataParameter));
                         }
+
+                        // Pass Object element.
+                        if (isListString(dataParameter)) {
+                            console.log("render list<string> input");
```
Comment (jmjuanes) [inline src/webcomponents/api/rest-endpoint.js:161]: Please remove this `console.log`.

File: src/webcomponents/api/rest-endpoint.js [hunk 3]
```diff
@@ -151,6 +155,13 @@ export default class RestEndpoint extends LitElement {
...
+                            bodyElements.push(this.buildListString(dataParameter));
+                        }
+
```
Comment (jmjuanes) [inline src/webcomponents/api/rest-endpoint.js:164]: Please remove this empty line.

File: src/webcomponents/api/rest-endpoint.js [hunk 4]
```diff
@@ -310,6 +322,26 @@ export default class RestEndpoint extends LitElement {
...
+            save: value => {
+                // stringToList
+                UtilsNew.setObjectValue(this.valuesTolist, fieldName, value?.split(","));
+                return value;
+            },
+            display: {
+                helpMessage: dataParameter.description
+            }
+        };
+
```
Comment (jmjuanes) [inline src/webcomponents/api/rest-endpoint.js:341]: Same, empty line.

**Review by jmjuanes — CHANGES_REQUESTED** [review]

**Review by jmjuanes — APPROVED** [review]

**Review by jmjuanes — APPROVED** [review]

**Pull request merged**